frontend/src/containers/base/MyPage.js

```diff
@@ -3,9 +3,16 @@ import GroupList from '../../components/group/GroupList';
 import Calendar from '../../components/base/Calendar';
 import ContentList from '../../components/content/ContentList';
 
+import './MyPage.scss';
+
 const MyPage = () => {
   return (
     <>
+      <div className="user__info">
+        <h1> My Page </h1>
+        <div className="user__info__email">email</div>
+        <div className="user__info__username">username</div>
+      </div>
```

Comment:
@cw10108 How about changing class name to "mypage" and others to "mypage__email", "mypage__username"

Reply:
> @cw10108 How about changing class name to "mypage" and others to "mypage__email", "mypage__username"

I left it user__info because user information+groups+calendar+fav content is my page. how about "my__info"?